server/db/database.sql

```diff
@@ -2,10 +2,11 @@ CREATE TABLE posts (
     id SERIAL PRIMARY KEY,
     title VARCHAR(255),
     pseudonym VARCHAR(255),
-    body VARCHAR(255)
+    body VARCHAR(255),
+    date DATE
 );
 
-INSERT INTO posts(title, pseudonym, body)
+INSERT INTO posts(title, pseudonym, body, date)
 VALUES
-    ('test title', 'batman', 'this is a test title'),
-    ('test title2', 'robin', 'this is another test title');
+    ('test title', 'batman', 'this is a test title', '2021-12-08'),
```

Comment:
the test seeds could have been left out

Reply:
true, it was used to see the routes and dates would work